PlaygroundAppUITests/XCUIApplication+StatusBarTests.swift

```diff
@@ -0,0 +1,44 @@
+import XCUITestHelpers
+import XCTest
+
+class XCUIApplicationStatusBarTests: XCTestCase {
+
+    let app = XCUIApplication()
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+        app.launch()
+    }
+
+    func testTapStatusBar() throws {
+        let getFirstListElement = { self.app.staticTexts["a"].firstMatch }
+        let getLastListElement = { self.app.staticTexts["z"].firstMatch }
+
+        // Ensure the list is in the default initial position by checking that the first element is
+        // hittable and the last isn't (meaning it's offscreen).
+        XCTAssertTrue(getFirstListElement().isHittable)
+        XCTAssertFalse(getLastListElement().isHittable)
+
+        var attempts = 0
+        let attemptsThreshold = 5
+        while getLastListElement().isHittable == false && attempts < attemptsThreshold {
+            app.swipeUp()
+            attempts += 1
+        }
```

Comment:
I can see another utility sprouting from here, ideally with better logic to compute whether it's at the bottom. An `attempts` counter of arbitrary threshold is not robust.

Reply:
At least now, I can't think of a better way. If `ScrollView` was used instead of the `List`, it would then be possible to wait for the vertical scroll bar to gain `100%` value, and use this instead of waiting for the last element to become hittable.

But this does not solve the initial issue. In both cases, we still need to `scrollUp()`, and need a sort of "fuse", to avoid swiping endlessly. From my tests, setting the scroll value to 100% in ScrollView programmatically is impossible (immutable), and also "incorrect", since the user is unable to do this. 🤷